capi/Makefile.am

```diff
@@ -17,6 +17,7 @@ libyami_capi_ldflags = \
 
 libyami_capi_cppflags = \
 	$(LIBVA_CFLAGS) \
+	-I$(top_srcdir)/interface \
```

Comment:
Hi @chivakker , 
@jsorg71 have good point. It's little mess since we install include from diffrent place.
like VideoEncoderCapi.h from capi/  amiVersion.h from common/
Maybe we can group all interface file to libyami project's include/interface  directory and  install to include/libyami. This will not break user.